examples/ethereum/ethereum.go

```diff
@@ -7,114 +7,90 @@ package ethereum
 import (
 	"context"
 	"math/big"
-	"strings"
 
-	"github.com/nakji-network/connector"
 	"github.com/nakji-network/connector/chain/ethereum"
 	"github.com/nakji-network/connector/common"
 	"github.com/nakji-network/connector/examples/ethereum/chain"
 	"github.com/nakji-network/connector/kafkautils"
 
 	"github.com/ethereum/go-ethereum/core/types"
-	"github.com/ethereum/go-ethereum/ethclient"
 	"github.com/rs/zerolog/log"
 )
 
 type Connector struct {
-	*connector.Connector // embed Nakji connector.Connector into your custom connector to get access to all its methods
+	*ethereum.Connector // embed Nakji ethereum.Connector into your custom connector to get access to all its methods
```

Comment:
It is confusing..

Reply:
do you mean `ethereum` is confusing? how about `evm`?

Reply:
`evm` looks good